project-addons/custom_emails_company/data/email_aviso_de_envio.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+
+        <!--Email template -->
+        <record id="email_template_edi_send_aviso_custom" model="mail.template">
```

Comment:
El comentario anterior aplicaría a esta pantilla más, las anteriores igual las queréis enviar manualmente desde un botón ¿no?

Reply:
Si, las queremos enviar manualmente. Sin embargo esta plantilla es la misma que comentas esta en stock_custom(action_donde), por lo que la he borrado de mi codigo. Lo unico es que le he agregado a la del stock_custom la referencia al archivo adjunto.